### PR TITLE
Implement event.detail

### PR DESCRIPTION
**Pull request by incrediblink**
- [x] Get event detail
- [ ] Get event list
- [ ] Get the pending news of an event
- [ ] Subscribe an event
- [ ] Create the header image
- [ ] Update the header image
- [ ] Create news
- [ ] Submit news
- [ ] Create the event
- [ ] Submit the event
- [ ] Update the event

### CODE REVIEW

**Comment by vincentdchan** [comment]
什么意思？没看懂

**Comment by incrediblink** [comment]
@vincentdchan 现在只做了一个获取单个事件信息的功能，但是可以先 merge，因为之前的 PR 有点 type…

**Comment by vincentdchan** [comment]
要不我们先把 API 那个 branch 给 merge 到 master 里面去，然后再基于 master 你完善这些 API

File: api/controllers/EventController.js [hunk 1]
```diff
@@ -6,5 +6,23 @@
  */
 
 module.exports = {
+  getEvent: async (req, res) => {
+    let name = req.param('name');
+    let event = await Event.findOne({
+      or: [
+        { id: parseInt(name, 10) > -1 ? parseInt(name, 10) : -1 },
```
Comment (vincentdchan) [inline api/controllers/EventController.js]: `parseInt` 默认十进制，不需要第二个参数

**Pull request merged**